src/191/main/kotlin/org/rust/cargo/runconfig/buildtool/Utils.kt

```diff
@@ -36,6 +39,27 @@ val ExecutionEnvironment.cargoPatches: MutableList<CargoPatch>
 
 private val CARGO_PATCHES: Key<MutableList<CargoPatch>> = Key.create("CARGO.PATCHES")
 
+private val ExecutionEnvironment.executionListener: ExecutionListener
```

Comment:
There is much common code between `191/.../Utils.kt` and `192/.../Utils.kt`. Let's extract it to something like `main/.../CommonUtils.kt`.